.gitlab/kitchen_testing/new-e2e_testing.yml

```diff
@@ -48,7 +42,7 @@
     EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --arch $E2E_ARCH --flavor $FLAVOR
   parallel:
     matrix:
-      - START_MAJOR_VERSION: [5, 6, 7]
+      - START_MAJOR_VERSION: [6]
         END_MAJOR_VERSION: [7]
```

Comment:
In the current situation it will test that the agent can be upgraded from agent 6 to agent 7. Is that what we want? Or maybe we want to keep only the test that try to upgrade agent 5 or agent 6 to agent 6?

Reply:
Yes I wanted to keep this because I thought it would make sense to ensure we can still upgrade from 6 to 7.